test/cli/tarantool_tcp.c

```diff
@@ -64,8 +64,7 @@ test_connect_tcp() {
 	header();
```

Comment:
README.rst mentions `test/tarantool-tcp.c`, so it worth to fix the path in the README.rst.

Reply:
Fixed in the last commit.